### DISCUSSION TITLE
Questions about batchsize

### DISCUSSION THREAD

**Discussion by machuanfeng** [BindsNET/bindsnet#620] [Q&A] [answered]
Hello, author, is the original intention of bindsnet design only to run with batchsize=1? I tried to increase batch_size, and there were some column errors.

**Comment by Hananel-Hazan** [comment, marked as answer]
Hi, the branch size mostly effect the PyTorch side of things. In BindsNET, you will need to change the network input shape and some other small parts. Please take a look in [this example ](https://github.com/BindsNET/bindsnet/blob/master/examples/mnist/batch_eth_mnist.py) on how to use batch size.

**Reply by machuanfeng** [reply]
Very helpful, thank you very much.
